### ISSUE TITLE
Decharter Inclusivity Working Group

### ISSUE THREAD

**Issue by jasnell** [nodejs/TSC#218]
Now that the Community Committee is officially underway, it's time for the TSC to consider officially dechartering the Inclusivity Working Group and archiving the repository.

/cc @nodejs/community-committee @nodejs/tsc

**Comment by bnoordhuis** [comment]
What are the actionables?  PR and vote at the next TSC meeting?

**Comment by joshgav** [comment]
We had someone from the Inclusivity WG contributing to the TSC; it might be helpful to invite someone from the CC to participate too, even though the CC doesn't report to the TSC like the Inclusivity WG did.

**Comment by nebrius** [comment]
I was thinking that I would sorta transition my role of representing the Inclusivity WG on the TSC (such as it is now) to representing the CC on the TSC. I think it's really important that we have good cross-communication between the two committees, and having people from each attend the other is important and useful.

I'm in favor of dechartering, and I don't think there's any reason to wait on it. It's already nonfunctional in practice.

I'm torn about archiving this repo though. On the one hand, there will be a new Inclusivity WG under the CC at some point in time, and we may want to reuse parts or all of the existing charter here for that, along with other documentation. At the same time, it may help the WG reboot to start from scratch so that we're not influenced and drawn towards old processes that contributed to the WG being ineffective.

**Comment by jasnell** [comment]
@bnoordhuis ... yes, and PR then a vote would be required.

@nebrius ... archiving the repository is neither permanent or destructive. We can choose to resurrect it at any time should it become useful and no part of the content existing there already would go away.

**Comment by nebrius** [comment]
@jasnell that's a good point. Ok yeah, I'm on board with archiving.

**Comment by jasnell** [comment]
This has been done.

**Comment by Trott** [comment]
The de-chartering is not reflected at https://github.com/nodejs/inclusivity or at https://nodejs.org/en/about/working-groups/.  Any reason not to?

**Comment by jasnell** [comment]
No, those should be updated also.

**Comment by hackygolucky** [comment]
Hey y'all, just as a thought exercise here: when you're considering dechartering a group, it would help to file in issue in that working group's repo. I had to dig to find this after I got an email that the Inclusivity WG team had been deleted,  even though plans had been discussed with folks who would like to (in the near future) work on rebooting Inclusivity in the future about -what- to do with it. I know @nebrius was speaking in good faith but I don't think that represents all of the folks interested in helping. 

@Trott did a great thing with the PR filed to archive(and the other way I noticed this happening). It would have been helpful for CommComm to discuss this with y'all to find out all dechartering entails so we could take steps where needed for those who would like to move forward. 

I suppose this is all the more reason that we should be making sure that reps from TSC/CommComm are attending each other's meetings and communicating joint concerns.

**Comment by Trott** [comment]
@hackygolucky Sorry about that. (I'm the one that deleted the team after emailing TSC, and waiting 24 hours-ish while getting a couple of :+1:'s.) In addition to your suggestions, I think we need to more explicitly document the steps that happen when WG's get dechartered. (Rename the repo? Is there a standard we have? Time frames, who to notify, etc.) On the other hand, except for the team deletion, all the other stuff that's happened is easily reversible via git, so if I or someone else messes up, it shouldn't be *too* bad to recover.

The inclusivity team had ten members. I'm pretty sure we can piece together who was on it if need be, perhaps via GitHub audit logs, but I'm going to surmise that the (much more easily recoverable) list of WG members in the README is probably more valuable.

**Comment by hackygolucky** [comment]
There are plans to reboot inclusivity, renaming the old repo nodejs/archived-inclusivity, and idea was to first get a much larger representation and perspective of folks with establishing CommComm and building from scratch the Inclusivity WG under CommComm scope. We don't want to blow away the history of the old repo if anyone would care to reference it, but I think we want to make it clear it is an almost entirely new effort when it is established

**Comment by Trott** [comment]
@hackygolucky FWIW I don't think anyone has suggested deleting the old repo, or at least not without being told immediately that We Don't Do That Sort Of Thing. (I might have suggested it once and had @jasnell respond with "oh no no no no no" within 14 nanoseconds.)

**Comment by hackygolucky** [comment]
@Trott yeah, I think all of this is reversible, and I don't think this ruffles any feathers. I guess I'm preoptimizing for other decharters which I think is pretty unlikely.

**Comment by Trott** [comment]
Yeah, decharters are infrequent, but it would be good to have a checklist of what to do , when to do it, who to notify, and how to notify them. The problem is creating such a thing for infrequent events is hard. Easy to miss stuff if you're not actually doing it.

**Comment by nebrius** [comment]
How about this: let's keep track of all actions we've taken so far to decharter in this thread, and we can create a checklist from that.

Off the top of my head, so far:

- The TSC voted to decharter
- Marked that The Inclusivity WG has been dechartered in https://github.com/nodejs/TSC/blob/master/WORKING_GROUPS.md#de-chartered-working-groups
- Updated the Inclusivity WG README to indicate it has been dechartered (In Progress)
- Deleted the GitHub team associated with the WG

Once again off the top of my head, we also need to:

- Remove the inclusivity@iojs.org mailing list

**Comment by Trott** [comment]
@nebrius Add:

* update the website repo to remove the working group from the website

**Comment by nebrius** [comment]
Updated (and switched to a checklist format), thanks @Trott!

**Comment by Trott** [comment]
Not sure if you want me editing your checklist or not but:

* email thing is in progress: https://github.com/nodejs/email/pull/39

* website thing is done: https://github.com/nodejs/nodejs.org/pull/1213

**Comment by nebrius** [comment]
I went ahead and updated, but feel free to edit as we go forward!
